Change the name from tidy5 back to tidy.
https://github.com/htacg/tidy-html5/issues/46

Change the mandir from man to share/man.
https://github.com/htacg/tidy-html5/issues/193
--- CMakeLists.txt.orig	2015-04-08 11:54:30.000000000 -0500
+++ CMakeLists.txt	2015-04-09 15:36:38.000000000 -0500
@@ -2,7 +2,7 @@
 # Prepare for changing the name to 'tidy'
 cmake_minimum_required (VERSION 2.8.8)
 
-set(LIB_NAME tidy5)
+set(LIB_NAME tidy)
 
 project (${LIB_NAME})
 
@@ -223,7 +223,7 @@
             VERBATIM
         )
 
-        install(FILES ${CMAKE_BINARY_DIR}/${TIDY_MANFILE} DESTINATION man/man1)
+        install(FILES ${CMAKE_BINARY_DIR}/${TIDY_MANFILE} DESTINATION share/man/man1)
     endif ()
 endif ()
 
